setup.py

```diff
@@ -1,9 +0,0 @@
-from setuptools import setup, find_packages
-setup(name='flightstatus',
-      version='0.0.1',
-      description='Predict flight delay and cancellation',
-      author='Hanh Nguyen',
-      author_email='myhanh.nguyen1211@gmail.com',
-      packages=find_packages("dev"),
-      package_dir={"": "dev"},
```

Comment:
Nitpick: since the rest of the repo uses single quotes, change these to single-quoted strings (instead of double-quoted strings).